lms/static/js/financial-assistance/views/financial_assistance_form_view.js

```diff
@@ -101,6 +103,28 @@
 
                 setExtraData: function(data) {
                     return _.extend(data, this.user_details);
+                },
+
+                validateCountry: function() {
+                    var $submissionContainer = $('.submission-error'),
+                        $errorMessageContainer = $submissionContainer.find('.message-copy'),
+                        $countryLabel = $('#user-country-title'),
+                        errorMsg = [
+                            "Please go to your <a href=\"",
+                            this.context.account_settings_url,
+                            "\">profile page</a> and provide your country of residence."
+                        ];
+
+                    if( !this.model.get('country') ){
+                        $countryLabel.addClass('error');
+                        $errorMessageContainer.append(
+                            "<li>" +
+                            gettext(errorMsg.join('')) +
```

Comment:
@AlasdairSwan @peter-fogg Is there a better way to create this string so that translators don't have to see the anchor tag? And we don't have to retranslate this string if the url ever changes?